test/test.lua

```diff
@@ -2321,6 +2321,49 @@ function rnntest.MaskZero()
    end
 end
 
+function forwardbackward(module, criterion, input, expected)
```

Comment:
can you make this function `local` to the file? Otherwise, it's global to every package!